src/main/java/com/synopsys/integration/blackduck/signaturescanner/command/ScanPaths.java

```diff
@@ -36,6 +39,26 @@ public ScanPaths(final String pathToJavaExecutable, final String pathToOneJar, f
         this.managedByLibrary = managedByLibrary;
     }
 
+    public List<String> createCommandForScannerVersion() {
+        final List<String> cmd = new ArrayList<>();
+        addScanCommandPrefix(cmd);
+        addScanCommandSuffix(cmd);
+        cmd.add("--version");
+
+        return cmd;
+    }
+
+    public void addScanCommandPrefix(final List<String> cmd) {
```

Comment:
I don't love the name of these methods. I assumed prefix meant adding something to the prefix of each command sent in. I don't have a great alternative, but something like addStandardCommandOptions.